heat/core/operations.py

```diff
@@ -166,6 +168,51 @@ def allclose(x, y, rtol=1e-05, atol=1e-08, equal_nan=False):
     return torch.allclose(x._tensor__array, y._tensor__array, rtol, atol, equal_nan)
 
 
+def any(x, axis=None, out=None):
+    """
+    Returns a boolean indicating if any elements are non-zero.
+
+    Parameters:
+    -----------
+    x : tensor
+        The tensor on which any should be calculated
+    axis : int, optional
+        Defines along which axis any should be calculated. If not specified, any will be computed on the whole
+        tensor.
+    out : tensor, optional
+        An output tensor into which the result will be copied. Result is a tensor with dtype=uint8/bool.
+
+    Returns:
+    --------
+    boolean_tensor : tensor of type uint8/bool
```

Comment:
type is ht.bool (even if internally a uint8). Please update the doc string accordingly